packages/compass-e2e-tests/tests/connection.test.ts

```diff
@@ -210,11 +210,10 @@ async function assertCannotCreateDb(
 
   // open the create database modal from the sidebar
   if (TEST_MULTIPLE_CONNECTIONS) {
-    await browser.clickVisible(
-      Selectors.sidebarConnectionActionButton(
-        connectionName,
-        Sidebar.CreateDatabaseButton
-      )
+    await browser.selectConnectionMenuItem(
```

Comment:
Cleaned this up because selectConnectionMenuItem() now works for the items that aren't collapsed too.